node_modules/oae-core/topnavigation/js/topnavigation.js

```diff
@@ -41,7 +65,19 @@ define(['jquery', 'oae.core', 'activityadapter'], function($, oae, ActivityAdapt
          */
         var initLeftMenu = function() {
             var $template = oae.data.me.anon ? $('#topnavigation-left-anonymous-template') : $('#topnavigation-left-loggedin-template');
-            oae.api.util.template().render($template, null, $('#topnavigation-left', $rootel));
+            oae.api.util.template().render($template, {
+                'hidesignup': widgetData.hidesignup
+            }, $('#topnavigation-left', $rootel));
+
+            if (authStrategyInfo.hasSingleExternalAuth) {
+                $('#topnavigation-left-signup').click(function() {
```

Comment:
Yes, if the user visits the tenant landing page anonymously, then there will be a Sign Up link still.